dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -6,6 +6,10 @@
 # the final image because of permission changes.
 FROM {{ .buildFrom }} AS home
 
+# Contains the elastic agent image variant, an empty string for the standard variant
+# or "offline" for the offline one.
+ENV ELASTIC_AGENT_IMAGE_VARIANT={{.Variant}}
```

Comment:
What do you think about omitting this from the environment when `.Variant` is empty?

Reply:
I'm neutral on it. I do kind of like having it empty because it makes it maybe more explicit that this is the "" variant.